spec/features/cards_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'filling the form' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.